YouVsVirus/Assets/Scenes/leveldisco/StartScreenLeveldisco_2.unity.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
Is this scene or meta file still in use?

Reply:
The mask itself does not know how many masks have been collected. And it may also change towards the other direction.
Example: The player collects 8 masks -> the counter is 8.
The player gets to the hospital and "gives it masks to the hospital" -> the player now has 0 masks -> the counter is 0.
However, the mask object itself cannot know that the player was at the hospital or not.
So the text is linked to the player´s inventory.

Reply:
Yes. Its the Disco screen that tells the player "Jeff is your best friend..."